### DISCUSSION TITLE
<Ygithub account suspended.>

### DISCUSSION THREAD

**Discussion by lokeshwaran17** [community/community#133260] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello,

on 19/07/2024, My github account has been suspended and I am reaching out to support via email for 5 days but there is no response. The github account has everything which I have achieved in tech till now.

Deleted account username - lokeshwaran173
Ticket ID - 2889699

I can't lose the account. I request anyone to please suggest what should I do in this case?

**Comment by LiteBrite82** [comment]
Hi @lokeshwaran17 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
